code/modules/spells/spell_types/godhand.dm

```diff
@@ -14,9 +14,9 @@
 	throw_range = 0
 	throw_speed = 0
 
-/obj/item/weapon/melee/touch_attack/New(var/spell)
-	attached_spell = spell
-	..()
```

Comment:
This was running code before parent New() that would now happen after New().

Does this matter in this case?